modules/sdk/src/main/scala/org/tessellation/sdk/infrastructure/gossip/GossipDaemon.scala

```diff
@@ -43,7 +43,8 @@ object GossipDaemon {
 
     private val logger = Slf4jLogger.getLogger[F]
 
-    implicit private val signedRumorOrdering: Ordering[Signed[Rumor]] = Signed.order[Rumor].toOrdering
+    implicit private val signedRumorOrdering: Ordering[Signed[PeerRumorBinary]] =
```

Comment:
This `Ordering` instance is no longer used. I will remove it from this PR with code-review fixes.